base/debug/stack_trace_unittest.cc

```diff
@@ -157,7 +162,8 @@ TEST_F(StackTraceTest, DebugOutputToStreamWithNullPrefix) {
 #endif  // !defined(__UCLIBC__) && !defined(_AIX)
 
 #if BUILDFLAG(IS_POSIX) && !BUILDFLAG(IS_ANDROID)
-#if !BUILDFLAG(IS_IOS)
+// Starboard does not support relative file paths, needed by |SpawnChild()|.
+#if !BUILDFLAG(IS_IOS) && !defined(USE_HACKY_COBALT_CHANGES)
```

Comment:
If it's not supported by Starboard, we should use !defined(STARBOARD)?
I'm not so sure when should we use this `USE_HACKY_COBALT_CHANGES`, @andrewsavage1 to clarify this?

Reply:
I added `USE_HACKY_COBALT_CHANGES` for changes we want to clean up later but want to get working now